homeassistant/components/fritz/manifest.json

```diff
@@ -2,7 +2,11 @@
   "domain": "fritz",
   "name": "AVM FRITZ!Box Tools",
   "documentation": "https://www.home-assistant.io/integrations/fritz",
-  "requirements": ["fritzconnection==1.10.3", "xmltodict==0.13.0"],
+  "requirements": [
+    "fritzconnection[qr]==1.10.3",
+    "xmltodict==0.13.0",
+    "segno==1.5.2"
```

Comment:
Why do we need to pin the `segno` requirement? `fritzconnection` depends on it.

Reply:
I noticed on a fresh container that is not automatically installed; didn't investigate any further honestly.

Simone

Reply:
https://fritzconnection.readthedocs.io/en/1.10.3/sources/library_modules.html?highlight=segno#example-create-a-qr-code-for-wifi-access

Reply:
I created a fresh dev container, applied the PR and verified that it doesn't work. Manually installed "segno" and is fine again.
Please notice the link above: docs says it won't be installed automatically by upstream lib.

Simone

Reply:
Since we specify `qr` as extra requirement pip should install `segno` automatically.

https://github.com/kbr/fritzconnection/blob/aaf45bd25e7b1f0d3221ff29b7d0e14b7153bcc1/setup.py#L52

Reply:
Humm, true. But anyway will work only for new install. What about users on 2022.11.x that already have fritzconnection 1.10.3 installed but without the [qr] ?

Simone

Reply:
Fore core installation users `pip` should handle this. Otherwise it's a `pip` problem. For other users that use our container all requirements are included in the container image.

Reply:
Removed pinning.
Most probably issue was due to wrong requirements for `fritz_callmonitor` installed before `fritz`.

Simone